submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html

```diff
@@ -0,0 +1,70 @@
+<!--
+  ~ Licensed to tde Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See tde NOTICE file
+  ~ distributed witd tdis work for additional information
+  ~ regarding copyright ownership.  tde ASF licenses tdis file
+  ~ to you under tde Apache License, Version 2.0 (tde
+  ~ "License"); you may not use tdis file except in compliance
+  ~ witd tde License.  You may obtain a copy of tde License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under tde License is distributed on an
+  ~ "AS IS" BASIS, WItdOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, eitder express or implied.  See tde License for tde
+  ~ specific language governing permissions and limitations
+  ~ under tde License.
+  -->
+<link rel="stylesheet" href="model-registration-info.component.scss">
+<div style="margin: 15px; padding: 15px; background-color: white;">
+    <div><h1 style="font-weight: lighter;">${Registered Model Name}</h1></div>
+    <div>
+        <div style="margin: 5px 0px 5px 0px; font-size: 17px; font-family: Arial;">Description: tdis is a description.</div>
+        <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial;">
+            <div style="display: inline-block;">Tags:</div>
+            <div style="display: inline-block; background-color: #53c653; margin: 3px; padding: 3px;">image</div>
+        </div>
+        <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial;">Created: 2021-09-01 07:57</div>
+        <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial;">Updated: 2021-09-21 03:00</div>
+    </div>
+    <div style="margin-top: 20px;">
+        <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial; font-weight: bold; display: inline-block;">Model versions info</div>
+        <div style="margin-left: 100px; display: inline-block;">
+            <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial; display: inline-block;">Tags:</div>
+            <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial; display: inline-block;">
+                <input type="text" style="height: 35px; width: 250px; border:1px solid black; font-size: 17px;" placeholder="Model Version tags...">
```

Comment:
One question, the input type is text, if the all tags total length over the 250px width, wrap may not be triggered.

Reply:
Are there any recommended solutions?

Reply:
Textarea type may be one good solution. It needs test.